Documentation/policy/language.rst

```diff
@@ -1205,6 +1209,10 @@ considered external to the cluster.
       - toFQDNs:
           - matchName: "www.google.com"
 
+A known issue with kube-apiserver policy matching is that the feature doesn't
+work in tunneling mode. See `GH-18049
+<https://github.com/cilium/cilium/issues/18049>`_.
```

Comment:
This text is in the "deny policies" section. Would it make sense to create a dedicated small section for example kube-apiserver policies to explain, and add this limitation there?

This would probably make sense as a sub-section under  the `.. _Entities based:` section.

Reply:
Ah thanks, I did not notice this was under a specific section, figured it was general. Fixed.